rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/db/RMDBManager.java

```diff
@@ -287,13 +288,13 @@ public void close() {
 
     public boolean addNodeSource(final NodeSourceData nodeSourceData) {
         try {
+            logger.info("Add node source " + nodeSourceData.getName() + " in database");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal " in database" 16 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)